attunement.lic

```diff
@@ -24,9 +25,13 @@ class Attunement
       return
     end
 
-    room_list = if @stationary_skills_only || DRStats.necromancer?
+    room_list = if @stationary_skills_only || (DRStats.necromancer? and !@force_power_walk)
```

Comment:
I like the idea of a waggle_set, but might name it something more generic. Would ROC/ROG be good enough to powerwalk?